_docs/setup/index.md

```diff
@@ -1,5 +1,108 @@
 ---
-title: Setup
+title: Forseti Installation
+order: 001
 ---
 
-TODO
\ No newline at end of file
+# {{ page.title }}
+
+This guide explains how to use the Forseti installation tool.
+
+## Before you begin
+
+Prior to running the setup wizard, you will need:
+
+  - A GCP organization for which you want to deploy Forseti.
+  - Org Admin IAM role in order for the script to assign the Forseti
```

Comment:
Expand acronym for first use: "Cloud Identity and Access Management (Cloud IAM)"